src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useRef } from 'react';
+import { createTodo, updateTodo } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+import { AuthContext } from '../Auth/AuthContext';
+
+type Props = {
+  todos: Todo[];
+  hasTodos: boolean;
+  handleLoadTodos: () => void;
+  query: string;
+  setQuery: (string: string) => void;
+  setIsEditing: (arg0: boolean) => void;
+};
+
+export const NewTodo: React.FC<Props> = ({
+  todos,
+  hasTodos,
+  handleLoadTodos,
+  query,
+  setQuery,
+  setIsEditing,
+}) => {
+  const newTodoField = useRef<HTMLInputElement>(null);
+  const user = React.useContext(AuthContext);
+
+  useEffect(() => {
+    if (newTodoField.current) {
+      newTodoField.current.focus();
+    }
+  }, []);
```

Comment:
you already have this in App component, just use it in one of them, on need to use it in both